mkstage4.sh

```diff
@@ -10,10 +10,20 @@ fi
 #set flag variables to null
 EXCLUDE_BOOT=0
 EXCLUDE_CONNMAN=0
+EXCLUDE_LOST=1
 QUIET=0
+USAGE="usage:\n\
+  `basename $0` [-q -c -b] [-s || -t <target-mountpoint>] <archive-filename> [custom-tar-options]\n\
+  -q: activates quiet mode (no confirmation).\n\
+  -c: excludes connman network lists.\n\
+  -b: excludes boot directory.\n\
+  -l: includes lost+found directory.\n\
```

Comment:
Actually, after taking a closer look I remember why I did it that way: getopts doesn't support long options. We can make `-l` mean "exclude lost+found", but if that's the default behavior then how would the user negate it? `+l`?

Here's a pattern I found somewhere for parsing long and short options. I didn't want to change the script this drastically without input, but we could do something like this:

```bash
while [ $# -gt 0 ]; do
    case "$1" in
        -d | --destination) destination="$2"; shift;;
        -n | --dry-run) dry_run=true;;
        -s | --summary) summary="$2"; shift;;
        --) shift; break;;
        -*) echo "Error: unrecognized option $1"; exit 1;;
        *) echo "Error: extra argument $1"; exit 1;;
    esac
    shift
done
```